mods/ra/weapons/missiles.yaml

```diff
@@ -173,193 +128,107 @@ MammothTusk:
 			Wood: 75
 			Light: 60
 			Heavy: 25
+			Concrete: 50
 		DamageTypes: Prone50Percent, TriggerProne, SmallExplosionDeath
-	Warhead@2Smu: LeaveSmudge
-		SmudgeType: Crater
-		InvalidTargets: Vehicle, Tank, Structure, Wall, Husk, Trees
 	Warhead@3Eff: CreateEffect
-		Explosions: med_explosion
 		ImpactSounds: kaboom12.aud
 		InvalidImpactTypes: Water, Air, AirHit
-	Warhead@4EffWater: CreateEffect
-		Explosions: small_splash
-		ImpactSounds: splash9.aud
-		ValidImpactTypes: Water
 	Warhead@5EffAir: CreateEffect
 		Explosions: med_explosion_air
 		ImpactSounds: kaboom25.aud
 		ValidImpactTypes: Air, AirHit
 
 Nike:
+	Inherits: ^AntiAirMissile
 	ReloadDelay: 15
 	Range: 7c512
 	Report: missile1.aud
-	ValidTargets: Air
 	Projectile: Missile
 		Arm: 3
-		Blockable: false
-		ContrailLength: 10
+		Inaccuracy: 0
 		Image: MISSILE
 		HorizontalRateOfTurn: 25
 		RangeLimit: 9c0
 		Speed: 341
 	Warhead@1Dam: SpreadDamage
-		Spread: 128
 		Damage: 50
 		ValidTargets: Air
 		Versus:
 			None: 90
 			Light: 90
 			Heavy: 50
-		DamageTypes: Prone50Percent, TriggerProne, SmallExplosionDeath
 	Warhead@3Eff: CreateEffect
 		Explosions: med_explosion_air
 		ImpactSounds: kaboom25.aud
 
 RedEye:
+	Inherits: Nike
 	ReloadDelay: 50
-	Range: 7c512
-	MinRange: 0c512
-	Report: missile1.aud
-	ValidTargets: Air
 	Projectile: Missile
-		Arm: 3
-		Blockable: false
-		ContrailLength: 10
-		Image: MISSILE
+		Inaccuracy: 0
 		HorizontalRateOfTurn: 20
-		RangeLimit: 9c0
 		Speed: 298
 	Warhead@1Dam: SpreadDamage
-		Spread: 128
 		Damage: 40
 		ValidTargets: Air
 		Versus:
 			None: 90
```

Comment:
`Nike` has the damage versus `None` set as `90` as well (so this can be removed).